scripts/solana_verify_through_off_chain_call.ts

```diff
@@ -0,0 +1,83 @@
+const anchor = require('@project-serum/anchor');
+const {Program} = require('@project-serum/anchor');
+const {PublicKey, SystemProgram} = require('@solana/web3.js');
+const {LeafValueCoder} = require('@umb-network/toolbox');
+
+const programId = "9agqAPFMkmekbTT4tcz8NCjL4WT2Ccpu8ayn1SGzVwC3";
+
+const IDL = JSON.parse(
+  require("fs").readFileSync("../artifacts/solana-idl/chain.json", "utf8")
+);
+
+const main = async() => {
+
+  const provider = anchor.AnchorProvider.env();
+  anchor.setProvider(provider);
+
+  let program = new Program(
+    IDL,
+    new PublicKey(programId),
+    provider
+  );
+
+  let blockId = 517349;
+  const seed = LeafValueCoder.encode(blockId, '');
+
+  const [blockPda, bump] = await PublicKey.findProgramAddress(
+    [seed], program.programId
+  );
+
+  const block = await program.account.block.fetch(blockPda);
+  const blockRoot = '0x' + Buffer.from(block.root).toString('hex');
+
+  console.log(`Block ID = ${blockId} root = ${blockRoot}`);
+  console.log(`timestamp = ${new Date(block.timestamp * 1000)}`);
+
+  let proofs = [
+    LeafValueCoder.encode("0x8aa4e4134178289504b4b6c7c85527b41905cf3d51ad95eaec44a87fbe773b82", 'FIXED_'),
```

Comment:
`LeafValueCoder` is not class for creating Buffers. please use `Buffer`.

Reply:
Currently using this in production for root:
`root.startsWith('0x') ? Buffer.from(root.slice(2), 'hex') : Buffer.from(root, 'hex')`